Formula/pulumi.rb

```diff
@@ -34,17 +35,14 @@ def install
       bin.install Dir["#{buildpath}/bin/*"]
       prefix.install_metafiles
 
-      # Install bash completion
-      output = Utils.popen_read("#{bin}/pulumi gen-completion bash")
-      (bash_completion/"pulumi").write output
+      # Install shell completions
+      system "#{bin}/pulumi gen-completion bash > pulumi.bash"
+      system "#{bin}/pulumi gen-completion zsh > _pulumi"
+      system "#{bin}/pulumi gen-completion fish > pulumi.fish"
```

Comment:
Why don't these work with the way all other formulae do this?

Reply:
what are the other formulae? I think this is the same way I did with `diesel`

Reply:
I don't think I've ever seen a formula do bash redirects in general, as far as I know it's always the way you're removing.